src/shared/services/education-organizations-service.ts

```diff
@@ -0,0 +1,57 @@
+import queryString from 'query-string';
+
+import { getEnv } from '../helpers';
+
+export interface ClientEducationOrganization {
+	// TODO move to typings repo
```

Comment:
Gereleased in typings 2.6.1.

Reply:
uw changes om 2.6.1 te ondersteunen zitten nog niet in release/v0.19.0 dus ik kan dit nog niet updaten